lib/ingestor-api/runtime/Dockerfile

```
@@ -8,7 +9,10 @@ RUN yum install -y git
 RUN python -m pip install pip -U
 
 COPY runtime/requirements.txt requirements.txt
-RUN python -m pip install -r requirements.txt "mangum>=0.14,<0.15" -t /asset --no-binary pydantic
+
+ARG PGSTAC_VERSION=0.9.5
```

```suggestion
ARG PGSTAC_VERSION
```

We shouldn't need this, right? Since we're passing it through build args?